### PR TITLE
Updated documentation

### PR DESCRIPTION
**Pull request by bastien70**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,7 +64,10 @@ Requires [Symfony CLI](https://symfony.com/download) and [Task](https://taskfile
 
 ## Configure the CRON job <a name="cron"></a>
 
-Configuring a CRON job allows you to automatically backup databases at the frequency of your choice.
+Configure the CRON task allowing you to activate the verification of backups to be made. **Set it to run every day**.
```
Comment (jmsche) [inline README.md:67]:
```suggestion
Configure the CRON task allows you to activate the verification of backups to be made. **Set it up to run every day**.
```

File: README.md [hunk 2]
```diff
@@ -64,7 +64,10 @@ Requires [Symfony CLI](https://symfony.com/download) and [Task](https://taskfile
...
+
+When you add databases, you can each time configure the periodicity of the backups to be made.
+Therefore, if you have configured a database so that a backup is made every week, the CRON task will check the current date and compare it with the date of the last backup of your database, in order to decide whether to establish a backup or not.
```
Comment (jmsche) [inline README.md:70]: Hmm IIRC the code does not do that, does it?

**Review by jmsche — CHANGES_REQUESTED** [review]
Please also update screenshots so your email address does not appear there

File: README.md (hunk 2 above)
Comment (bastien70) [reply README.md:70]: He does yes. When you add a database, you configure the backup frequency.

The user configures a single CRON task that he must launch, for example, every day.

Every day, the CRON task will check all the databases and check the date of the next backup. If the date corresponds to today's date, there is a backup.

Comment (jmsche) [reply README.md:70]: AFAICT the next date is already calculated when running the command, and the command retrieves databases based on the calculated next date, so the last sentence is wrong

Comment (bastien70) [reply README.md:70]: Ah yes you are right, I change that

**Review by jmsche — APPROVED** [review]

**Pull request merged**